docs/source/markdown/podman-create.1.md

```diff
@@ -364,6 +364,13 @@ The following example maps uids 0-2000 in the container to the uids 30000-31999
 
 Add additional groups to run as
 
+#### **--groups**=*false*
+
+Allow container to use the user's supplementary group access. If file systems or
+devices are only accessible by the rootless users group, this flag tells the OCI
+runtime to pass the group access into the container. Currently only available
+with the crun OCI runtime. (Not available for remote commands)
```

Comment:
Does this work as root? Does it make any sense as root?

Reply:
Maybe if we can specify the groups.  I don't know what it means to specify all groups.

```
# podman run --user=dan --groups=eng ...
```
Might make some sense.

Reply:
root in a user namespace would have the same issue and not able to map groups from the host (unless they are mapped in the user namespace)